src/main/java/seedu/duke/PassengerParser.java

```diff
@@ -0,0 +1,16 @@
+package seedu.duke;
+
+import java.util.Arrays;
+
+public class PassengerParser {
+    private static PassengerList passengers = new PassengerList();
+
+    public static void parse(String[] inputWords) {
+        String command = inputWords[1];
+        if (command.equalsIgnoreCase("add")) {
+            String[] passengerDetailsArray = Arrays.copyOfRange(inputWords, 2, inputWords.length);
+            String passengerDetails = String.join(" ", passengerDetailsArray);
+            passengers.addPassenger(passengerDetails);
```

Comment:
It would be good if we can create a new package caused commands to store all types of commands. 
i.e. 
1 abstract Command class to house all the execute method
1 class that inherits from Command for AddCommand to differentiate passenger add and flight add
1 class that inherits from Command for DeleteCommand to differentiate passenger delete and flight delete
1 class that inherits from Command for ListCommand to differentiate passenger list and flight list